### ISSUE TITLE
Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x7fa9a442b440> could not be transformed and will be executed 

### ISSUE THREAD

**Issue by PedroDKE** [tensorflow/tensorflow#34319]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution: Ubuntu 18.06
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below): 2.0.0
- Python version: 3.7.4
-using spyder3 ide in anaconda
- CUDA/cuDNN version: None

**Describe the current behavior**
getting this error and the training accuracy doesnt improve

WARNING:tensorflow:Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x7fa9a442b440> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
WARNING: Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x7fa9a442b440> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
**Describe the expected behavior**
well, increasing ccuracy and no error basically
**Code to reproduce the issue**

from tensorflow.keras.models import Sequential
from tensorflow.keras import layers
import numpy as np

y_train = np.random.rand(1)
epochs = 1
x_train = np.random.rand(1,20,16)

model = Sequential()
model.add(layers.LSTM(activation='sigmoid',units = 3,batch_input_shape = (1,20,16)))
model.add(layers.Dense(2, activation='softmax'))

model.compile(loss='sparse_categorical_crossentropy',
              optimizer='adam',
              metrics=['accuracy'])
model.summary()
history = model.fit(x_train, y_train, batch_size=1, epochs=epochs)

this reproduces the error in my usecase

edit: using the solution proposed in
https://github.com/tensorflow/tensorflow/issues/32377

did not work for me ( Requirement already satisfied: gast==0.2.2 )

**Comment by ravikyram** [comment]
@PedroDKE 

I tried reproducing the issue with TF 2.0 using colab. However i am not seeing any error message. Please, help me in reproducing the issue.Please, find the gist [here](https://colab.sandbox.google.com/gist/ravikyram/d03583e7c0604f9413ab84cd7ccf64f7/untitled378.ipynb). Thanks!

**Comment by PedroDKE** [comment]
wasn't able to reproduce it there either, i tried to reinstall tensorflow but it didtn help either.
here is the error with verbosity set to 10:

INFO:tensorflow:Converted call: <function TensorLikeDataAdapter.__init__.<locals>.permutation at 0x7f05a833dcb0>
    args: (<tf.Tensor 'args_0:0' shape=() dtype=int64>,)
    kwargs: {}

INFO:tensorflow:Whitelisted: <function TensorLikeDataAdapter.__init__.<locals>.permutation at 0x7f05a833dcb0>: DoNotConvert rule for tensorflow
INFO:tensorflow:Converted call: <function TensorLikeDataAdapter.__init__.<locals>.slice_batch_indices at 0x7f062b289dd0>
    args: (<tf.Tensor 'args_0:0' shape=(1,) dtype=int64>,)
    kwargs: {}

INFO:tensorflow:Whitelisted: <function TensorLikeDataAdapter.__init__.<locals>.slice_batch_indices at 0x7f062b289dd0>: DoNotConvert rule for tensorflow
INFO:tensorflow:Converted call: <function TensorLikeDataAdapter.__init__.<locals>.grab_batch at 0x7f050ac61200>
    args: (<tf.Tensor 'args_0:0' shape=(1,) dtype=int64>, (<tf.Tensor 'args_1:0' shape=(1, 20, 16) dtype=float64>, <tf.Tensor 'args_2:0' shape=(1, 1) dtype=float64>))
    kwargs: {}

INFO:tensorflow:Whitelisted: <function TensorLikeDataAdapter.__init__.<locals>.grab_batch at 0x7f050ac61200>: DoNotConvert rule for tensorflow

the error occurs when i try to fit the model

**Comment by jvishnuvardhan** [comment]
@PedroDKE I tried my laptop (local) and colab and could not reproduce the issue. Can you please share a gist which is throwing the error. Thanks!

Please close the issue if this was already resolved. Thanks!

**Comment by jvishnuvardhan** [comment]
I think this issue was resolved as I could not reproduce the issue in my local as well as colab. I am closing this issue but please feel free to reopen if the issue persists again. Thanks!

**Comment by jottbele** [comment]
Hi, I have the same problem.

**Comment by SadakVali** [comment]
Hi, I have the same problem.
